packages/one-bx-shell-app/src/app/components/plugin-launcher/plugin-launcher.component.ts

```diff
@@ -36,17 +36,13 @@ export type ExtendedPluginDescriptor = PluginDescriptor & {
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class PluginLauncherComponent implements OnChanges {
-  @Input()
-  type: Scope = Scope.Page;
-
+  /** plugin name */
```

Comment:
We will be moving this to App-shell

Reply:
Already, you have merged that PR in appshell
When it released we can delete this component